iiab

```diff
@@ -566,50 +566,56 @@ fi
 # OLD WAY ABOVE - fails w/ sev ISPs per https://github.com/iiab/iiab/issues/871
 
 # I. Start BitTorrent downloads, e.g. if /etc/iiab/local_vars.yml requested any
-if systemctl -q is-active transmission-daemon; then
+# odd this runs when kalite is not installed
+#if systemctl -q is-active transmission-daemon; then
+if $(systemctl -q is-active transmission-daemon) && ! $(ischroot -t); then
     echo -e "\n\nSTARTING BITTORRENT DOWNLOAD(S) for KA Lite...Please Monitor: http://box:9091\n"
     transmission-remote -n Admin:changeme -t all --start
 fi
 
 touch $FLAGDIR/iiab-complete
 
 # J. Educate Implementers prior to rebooting!
-echo -e "\n\n         ┌───────────────────────────────────────────────────────────┐"
-echo -e "         │                                                           │"
-echo -e "         │   INTERNET-IN-A-BOX (IIAB) SOFTWARE INSTALL IS COMPLETE   │"
-echo -e "         │                                                           │"
-echo -e "         └───────────────────────────────────────────────────────────┘\n"
+if ! $($MFABT); then
```

Comment:
Some people in IIAB's testing community use --risky but do _not_ want a reboot interrupting their workflow.

Should this be?

```suggestion
if ! $($MFABT); && ! $(ischroot -t); then
```

Reply:
> Some people in IIAB's testing community use --risky but do not want a reboot interrupting their workflow.

They would currently have to 'ctrl c' out of the reboot before answering the question at the end. This would bypass the question and exit. Could add a "IIAB Done" before the exit 0